tests/e2e/actor_features/actor_features_test.go

```diff
@@ -382,7 +382,7 @@ func TestActorFeatures(t *testing.T) {
 		req := actorReminderOrTimer{
 			Data:    "reminderdata",
 			DueTime: "1s",
-			Period:  "10s",
+			Period:  "20s",
```

Comment:
```suggestion
			Period:  "10s",
```

This changes what the test is testing. This is fixed in https://github.com/dapr/dapr/pull/8398

Reply:
thats a diff test, this diff is from the 3xx lines, the diff from the linked PR impacts the 1xx lines. I dont think its the same issue

Reply:
I think you really are changing the test behaviour though, and we shouldn't. Please revert.

Reply:
I increased the period bc there was flakiness in CI, but not locally. I believe the reminder was firing back due to the 10s and CI being slower, I put 20s just to give CI more time. The period being upped should not impact the intended behavior of the test, but accounts for a slower env in CI. I can revert it since you seem adamant about it, but if it is flaky then I suggest upping the time a little. I understand why you don't want it upped to 20s since we 'reset' it to the 20s